.github/workflows/ci.yml

```
@@ -14,24 +14,53 @@ jobs:
 
   build:
     name: Python==${{ matrix.python-version }} | ${{ matrix.django-version }}
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-24.04
```

We are explicitly mentioning the Ubuntu version in CI across all modules of OpenWISP. Let's keep that for consistency.

---

okk will change it to 24.04 in other modules , if i work on the` ci.yml` part.